src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -92,7 +92,10 @@ public static TypedData ToRpc(this object value)
                     Dictionary<string, object> parameters = (Dictionary<string, object>)routeData;
                     foreach (var pair in parameters)
                     {
-                        http.Params.Add(pair.Key, pair.Value.ToString());
+                        if (pair.Value != null)
```

Comment:
had to make this change to the RPC message conversion, as it currently does to accept/deals with a null value. Passing empty here would not be semantically the same and would have unintended side effects (e.g. in node, the actual property would be defined). We need to decide how we can pass values in "default" state as part of this payload